ql/src/test/org/apache/hadoop/hive/ql/TestTxnExIm.java

```diff
@@ -566,4 +567,28 @@ public void testMMExportAborted() throws Exception {
         TestTxnCommands2.stringifyValues(data), rs);
 
   }
-}
+
+  @Test public void testExportPartitionedOrcWithOutColumnStats() throws Exception {
+
+    runStatementOnDriver("drop table if exists T");
+    runStatementOnDriver("drop table if exists Tstage");
+    runStatementOnDriver("create table T (a int, b int) partitioned by (p int) stored"
+        + " as orc tblproperties('transactional'='true')");
+    //Tstage is the target table
+    runStatementOnDriver("create table Tstage (a int, b int) partitioned by (p int) stored"
+        + " as orc tblproperties('transactional'='true')");
+    //this creates an ORC data file with correct schema under table root
+    runStatementOnDriver("insert into Tstage values(1,2,10),(3,4,11),(5,6,12)");
+    final int[][] rows = { { 3 } };
+    //now we have an archive with 3 partitions
+    runStatementOnDriver("export table Tstage to '" + getWarehouseDir() + "/1'");
```

Comment:
Check the COLUMN_STATS_ACCURATE property is true on source table and partition before and after export operation to confirm we don't overwrite anything in their metadata.